src/nuspec/docfx.msbuild/docfx.msbuild.nuspec

```diff
@@ -9,6 +10,7 @@
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <licenseUrl>https://github.com/dotnet/docfx/blob/dev/LICENSE</licenseUrl>
     <projectUrl>https://github.com/dotnet/docfx</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
```

Comment:
Looks like this change does not reflect the purpose described in the title **developer dependency attribute**?